shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -118,7 +120,6 @@ @implementation FlutterEngine {
   std::shared_ptr<flutter::ThreadHost> _threadHost;
   std::unique_ptr<flutter::Shell> _shell;
   NSString* _labelPrefix;
-  std::unique_ptr<fml::WeakNSObjectFactory<FlutterEngine>> _weakFactory;
 
   fml::WeakNSObject<FlutterViewController> _viewController;
```

Comment:
And this one, in `setViewController` it can become `_viewController = viewController;`, and the `- (FlutterViewController*)viewController` getter implementation can be removed.

Reply:
Yep! This is patch is still WIP -- it's at the "just barely not leaking anymore stage". There's a ton of cleanup still to come.

Reply:
Added a TODO -- as discussed in person, this one is going to need to wait till `platform_view_ios.h` is migrated since we're going to need a `__weak` modifier on one of the C++ class members, and since that header is pulled into both FlutterEngine and FlutterViewController, we'd need to migrate FlutterViewController in this same patch.

Getting closer though!